### ISSUE TITLE
Hangs on login (Meizu PRO 7 Plus)

### ISSUE THREAD

**Issue by VoidAspect** [wireapp/wire-android#1440]
## The problem

The app hangs after entering credentials in login dialog. (Existing account)

## Environment

* Wire version that exhibits the issue: 3.7.398
* Last Wire version that did not exhibit the issue (if applicable):
* Android OS version used to run Wire: 7.0
* Is this a custom firmware or a stock one: Flyme 6.1.2.3G (stock)
* Mobile phone model/manufacturer: M793H (Meizu PRO 7 Plus)
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): All network types

## Details

The issue seems to be device/ROM specific. Here is how I reproduce it:

1. Open the app fresh after downloading
1. Go to LOG IN in menu
1. Enter my credentials

After these steps, app hangs (see screenshot). 
If I, afterwards, clear the cache and try launching Wire once again, it just displays black screen.
Note, that the same issue occurs when I try to login with phone number.

## Screenshots

![s80227-131802](https://user-images.githubusercontent.com/15218015/36977141-d190b9b0-2088-11e8-9d53-7506e013ad13.jpg)

**Comment by someoneEsle** [comment]
Hi, thanks for reporting.
Could you tell me a little more about your situation? Why are you trying to log in? Did the app log you out at some point, or are you trying to log in on a new device/phone that you never used before?

If you feel helpless, you can try clearing the *data* instead of the cache, and try to log in again (**WARNING**: this will delete your history on this device, if you have any)

**Comment by VoidAspect** [comment]
@someoneEsle 
Hello, thanks for getting back to me.

I'm trying to log in because it's my new device. Before, I used Wire versions for Android and Linux Desktop with no such issues.

Deleting data resets the whole thing to step 1, meaning that I see login screen again, but the app still hangs after I enter creds.

**Comment by someoneEsle** [comment]
Interesting. Are you able to log in on app.wire.com? I know it's a dumb question but I need to know if the password you're using is correct ^^
Also, does the spinning ever end? Does it ever reach a timeout? Or does it just keep spinning forever? If so, how long have you tried waiting?

**Comment by VoidAspect** [comment]
@someoneEsle 
Yes, I am indeed able to login on the web version.
No, the spinning doesn't seem to end. The longest I've been waiting was about 10 min.

**Comment by someoneEsle** [comment]
Have you tried rebooting your phone and log in right after that? Have you ever experienced similar connection issues with other apps?

**Comment by VoidAspect** [comment]
@someoneEsle Yes, I've tried rebooting and reinstalling even, to no avail. No, other apps don't have similar issues.

**Comment by dlfsdev** [comment]
Has there been any progress on this? I've just recently started having the exact same problem.

**Comment by msdhedhi** [comment]
I am also running into the same issue on the Android 9 x86_64 emulator. I am using the wire android app version 3.37.820 downloaded from playstore.
The ADB log shows the following error.

09-09 13:31:18.937  2681  2749 E com.wire: No implementation found for void com.waz.call.FlowManager.attach(android.content.Context, long) (tried Java_com_waz_call_FlowManager_attach and Java_com_waz_call_FlowManager_attach__Landroid_content_Context_2J)

I looked the source code for waz ( https://github.com/wireapp/wire-audio-video-signaling/blob/master/android/jni/flow_manager.cc and https://github.com/wireapp/wire-audio-video-signaling/blob/master/android/java/com/waz/call/FlowManager.java ) and it looks like the "attach" API at some point had just one parameter but now has 2 parameters. Could it be that the .so file in the package did not get updated???
